sdk/relay/Microsoft.Azure.Management.Relay/tests/Tests/ScenarioTests.cs

```diff
@@ -26,6 +29,9 @@ public partial class ScenarioTests
         private ResourceManagementClient _resourceManagementClient;
         private RelayManagementClient _relayManagementClient;
         private RecordedDelegatingHandler handler = new RecordedDelegatingHandler();
+        private KeyVaultManagementClient _KeyVaultManagementClient;
+        private NetworkManagementClient _NetworkManagementClient;
+        private ManagedServiceIdentityClient _IdentityManagementClient;
```

Comment:
@v-Ajnava , why are we including external sdks like KeyVault and Identity when RelayNamespace does not support Encryption and MSI

Reply:
Only NetworkManagementClient is sufficient, for private endpoints, remove the others.